<launch>
	<arg name="sensor_num" default="1"/>
	<node pkg="vl53l0x_driver" type="measure_proximity_node" name="measure_proximity_node" output="screen" >
		<param name="sensor_num" value="$(arg sensor_num)"/>
  	</node>
</launch>
